README.md

```diff
@@ -405,6 +406,9 @@ ethree.deleteGroup(id: groupId) { error in
 ## Double Ratchet Chat
 In this section, you'll find out how to create and use Double Ratchet chats feature.
```

Comment:
```suggestion
In this section, you'll find out how to create and manage secure chat sessions between two users using the Double Ratchet algorithm so that each message is separately encrypted.
```

Reply:
Don't we have a Double Ratchet whitepaper? If so, can we link to that here?